x-pack/plugins/uptime/public/components/fleet_package/lazy_synthetics_policy_create_extension.tsx

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Create and Edit Lazy Components may need to be wrapped in some type of permissions logic.